src/Symfony/Component/Workflow/Event/Event.php

```diff
@@ -37,16 +37,25 @@ public function __construct($subject, Marking $marking, Transition $transition)
         $this->transition = $transition;
     }
 
+    /**
+     * @return Marking
```

Comment:
Yes, should be reverted. IDEs should be able to infer the return type from code (as the `$marking` property is private and is set during initialisation).